src/afl-fuzz.c

```diff
@@ -889,6 +894,8 @@ int main(int argc, char **argv_orig, char **envp) {
 
   }
 
+  if (unlikely(afl->afl_env.afl_statsd == 1)) { statsd_setup_format(afl); }
```

Comment:
I'd prefer a simple
`if (unlikely(afl->afl_env.afl_statsd)) { statsd_setup_format(afl); }`
for readability :)